forte/data/data_store.py

```diff
@@ -485,6 +485,48 @@ def _is_annotation(self, type_name: str) -> bool:
         entry_class = get_class(type_name)
         return issubclass(entry_class, (Annotation, AudioAnnotation))
 
+    def all_entries(self, entry_type_name: str) -> Iterator[List]:
+        """
+        Retrieve all entry data of entry type ``entry_type_name`` and
+        entries of subclasses of entry type ``entry_type_name``.
+
+        Args:
+            entry_type_name (str): the type name of entries that the User wants to retrieve.
+
+        Yields:
+            Iterator of raw entry data in list format.
+        """
+        for entry_type_key in self.__elements:
+            if entry_type_key == entry_type_name or self._is_subclass(
+                entry_type_key,
+                get_class(entry_type_name),
+            ):
```

Comment:
Can we create a `get_all_subclass` function here?

1. It will be used in multiple places
2. We can create a more efficient implementation later (you can leave the implementation now)
3. I think we need to sort the entry keys, otherwise the iteration order is kinda random